chapter_linear-networks/softmax-regression_vn.md

```diff
@@ -160,23 +160,27 @@ our outputs are given by a matrix-vector product of our weights by our inputs pl
 ### Softmax Operation
 -->
 
-### *dịch tiêu đề phía trên*
+### Hàm Softmax
 
 <!--
 The main approach that we are going to take here is to interpret the outputs of our model as probabilities.
 We will optimize our parameters to produce probabilities that maximize the likelihood of the observed data.
 Then, to generate predictions, we will set a threshold, for example, choosing the *argmax* of the predicted probabilities.
 -->
 
-*dịch đoạn phía trên*
+Cách tiếp cận chính mà ta sẽ thực hiện ở đây là diễn tả các kết quả của mô hình như một xác suất.
+Ta sẽ tối ưu hóa bộ tham số để đạt được xác xuất mà xảy ra đồng thời các dữ liệu quan sát được là cao nhất
+Sau đó, ta sẽ đưa ra dự đoán bằng cách đặt một ngưỡng, ví dụ chọn *argmax* của giá trị xác suất dự đoán.
```

Comment:
```suggestion
Sau đó, ta sẽ đưa ra dự đoán bằng cách đặt ngưỡng xác suất, ví dụ dự đoán nhãn đúng là nhãn có xác suất cao nhất (dùng hàm *argmax*).
```

Sao cái ví dụ có vẻ hông sát với cái đang nói nhỉ. Đặt ngưỡng 0.5 mà ra vector xác suất là: 0.4, 0.3, 0.2, 0.1 thì không có nhãn nào đạt ngưỡng hết nhưng argmax vẫn ra kết quả.

Reply:
Em nghĩ cái ví dụ lúc này chưa phải ví dụ cho softmax, mà chỉ đang dừng lại cho bài toán binary classification (kiểu dùng hàm sigmoid để phân loại), cho nên lúc sau tác giả mới đề cập đến vấn đề là kết quả của hàm logit không thật sự tốt để coi nó như là một phân phối xác xuất ạ.

Reply:
Mình nhận xét về cả câu văn luôn ấy. Theo cách hiểu của mình, khi đã nói về đặt ngưỡng thì ta sẽ cần một số cố định (ngưỡng) và một số thay đổi để so sánh với số cố định đó xem có quá ngưỡng hay chưa. Tác giả nói về đặt ngưỡng nhưng lại không ví dụ về một ngưỡng như vậy.

Ví dụ của tác giả là dùng argmax, tức là không cần chọn số cố định nào cả.

Reply:
à à em đã hiểu ý anh